src/bin/wasm2obj.rs

```diff
@@ -77,6 +77,7 @@ Options:
     --cache-compression-level=<compr_level>
                         enable caching system, use custom compression level for new cache, values 1-21
     --enable-simd       enable proposed SIMD instructions
+    --optimize          runs optimization passes on the translated functions
```

Comment:
Maybe `-O` like rustc?

Reply:
thanks for the suggestion. I added a new commit which adds -O.

Please carefully review it even though it's a trivial change because this is the first time I'm doing anything with rust...